sklearn/feature_extraction/text.py

```diff
@@ -435,6 +435,8 @@ def __init__(self, input='content', encoding='utf-8',
         self.stop_words = stop_words
         self.n_features = n_features
         self.ngram_range = ngram_range
+        if ngram_range[0] > ngram_range[1]:
+            raise ValueError("min_n must not be greater than max_n.")
```

Comment:
It might be also useful to print the actual values of `min_n`, `max_n` [as it is done for `max_features`](https://github.com/AngeloK/scikit-learn/blob/087f7d07ab0361f9f82c9a5a9387a37d23bdeaa0/sklearn/feature_extraction/text.py#L685)